docs/debuggers_python.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Python Debuggers in Tilt
+layout: docs
+---
+
+Debuggers are an invaluable part of many development workflows. When you're running your app(s) in Kubernetes, using debuggers suddenly gets much harder, but Tilt's port forwarding functionality makes it easy to use any debugger that exposes a port for external connect.
+
+At the current moment, our recommended Python debugger for use with Tilt is [`remote-pdb`](https://pypi.org/project/remote-pdb/) (we will list other debugger options here as we validate and write guides for them).
+
+## Debugging your project with Tilt and a remote debugger
+
+I'll be referring to [this example repo](https://github.com/windmilleng/debugger-examples/tree/master/python/remote-pdb), which is configured to use `remote-pdb`. Clone it and follow along!
```

Comment:
```suggestion
We'll use [this example repo](https://github.com/windmilleng/debugger-examples/tree/master/python/remote-pdb), which is configured to use `remote-pdb`. Clone it and follow along!
```

Reply:
😅I LIKE USING LOTS OF WORDS THEY MAKE ME SOUND SMART

Reply:
Hah!

Seriously though, this "I will be doing" instead of "I will do" construction that you see in English all the time... it sounds *so* weird to non-native speakers.